### ISSUE TITLE
Neat and organised file structure 

### ISSUE THREAD

**Issue by RunGT** [fac20/sp-bghm#50]

**Comment by aissshah** [comment]
Yes, was a pleasure to go through the file tree and everything was named in a standardised way. Although, I had a question about useLocation.js? Not sure I could find where else it was used..
